src/components/DataOwner.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+    Popover,
+    PopoverContent,
+    PopoverTrigger,
+} from "@/components/ui/popover"
+import {
+    Select,
+    SelectContent,
+    SelectGroup,
+    SelectItem,
+    SelectLabel,
+    SelectTrigger,
+    SelectValue,
+} from "@/components/ui/select"
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/avatar";
+import Typography from "@/components/ui/Typography"
+import { UserPlus } from "lucide-react";
+import User from "./User";
+
+type User = {
```

Comment:
type User defined here and in `User.tsx`

rename roll to "role"

I would not have initials as a field because it's double info, it can be a function that creates initials from name instead